Makefile

```
@@ -42,6 +42,9 @@ starknet-test: starknet-clean
 starknet-deploy: starknet-build
 	@./contracts/cairo/deploy.sh
 
+starknet-upgrade: starknet-build
+	@./contracts/cairo/upgrade.sh
+
 starknet-deploy-and-connect: starknet-build
```

```suggestion
starknet-deploy-and-connect:
```
Otherwise, starknet-deploy-and-connect builds twice

---

:heavy_check_mark: